diff --git a/node_modules/@d11/react-native-fast-image/android/build.gradle b/node_modules/@d11/react-native-fast-image/android/build.gradle
index e166ccf..c73cd8b 100644
--- a/node_modules/@d11/react-native-fast-image/android/build.gradle
+++ b/node_modules/@d11/react-native-fast-image/android/build.gradle
@@ -32,7 +32,7 @@ apply plugin: 'com.android.library'
 
 android {
     namespace "com.dylanvann.fastimage"
-    compileSdkVersion safeExtGet('compileSdkVersion', 28)
+    compileSdkVersion safeExtGet('compileSdkVersion', 30)
     buildToolsVersion safeExtGet('buildToolsVersion', '28.0.3')
     defaultConfig {
         minSdkVersion safeExtGet('minSdkVersion', 16)
@@ -91,3 +91,7 @@ dependencies {
     implementation "com.github.zjupure:webpdecoder:2.6.${glideVersion}"
     annotationProcessor "com.github.bumptech.glide:compiler:${glideVersion}"
 }
+
+tasks.named("generateCodegenArtifactsFromSchema") {
+    dependsOn(":@d11_react-native-fast-image:generateCodegenSchemaFromJavaScript")
+}
