diff --git a/node_modules/@react-native/gradle-plugin/react-native-gradle-plugin/src/main/kotlin/com/facebook/react/tasks/GeneratePackageListTask.kt b/node_modules/@react-native/gradle-plugin/react-native-gradle-plugin/src/main/kotlin/com/facebook/react/tasks/GeneratePackageListTask.kt
index 36cc66c..8ec80c8 100644
--- a/node_modules/@react-native/gradle-plugin/react-native-gradle-plugin/src/main/kotlin/com/facebook/react/tasks/GeneratePackageListTask.kt
+++ b/node_modules/@react-native/gradle-plugin/react-native-gradle-plugin/src/main/kotlin/com/facebook/react/tasks/GeneratePackageListTask.kt
@@ -43,8 +43,7 @@ abstract class GeneratePackageListTask : DefaultTask() {
 
     val packageName =
         model.project?.android?.packageName
-            ?: error(
-                "RNGP - Autolinking: Could not find project.android.packageName in react-native config output! Could not autolink packages without this field.")
+            ?: "com.betatech.reactnativebrownfiled"
 
     val androidPackages = filterAndroidPackages(model)
     val packageImports = composePackageImports(packageName, androidPackages)
